.circleci/config.yml

```diff
@@ -1,74 +1,298 @@
 version: 2.1
 
+
+executors:
+  py36:
+    docker:
+      - image: shivamshriwas/torchserve:py36
```

Comment:
Move this to ECR & Use Credentials as env variables - https://github.com/pytorch/pytorch/blob/master/.circleci/config.yml

Reply:
Moved docker images from dockerhub to AWS ECR